### DISCUSSION TITLE
Custom HTML in table rows?

### DISCUSSION THREAD

**Discussion by mitya33** [buefy/buefy#3621] [Q&A]
I'm sure this must be possible somehow but I can't see how. There's no slot for this, seemingly, and it seems any custom HTML in your row data is not parsed, just ends up as text.

**Comment by lownlazy** [comment]
try using v-html: <p v-html='yourhtmlcode'></p>
https://www.geeksforgeeks.org/vue-js-v-html-directive/

**Reply by mitya33** [reply]
How? That's a directive to go on a tag, but when generating a table in Buefy I don't control the actual `td` tags.

**Reply by adamleemiller** [reply]
You do not need to control the <td> tag. You can simply use something like <span v-html="data"></span> or even a div or something.

**Reply by mitya33** [reply]
Could you possibly show a simple example?
